static/app/views/starfish/queries/useIndexedSpans.tsx

```diff
@@ -15,15 +15,23 @@ interface Filters {
   [key: string]: string;
 }
 
-export const useIndexedSpans = (
-  filters: Filters,
-  sorts: Sort[] = [{field: 'timestamp', kind: 'desc'}],
-  limit: number = DEFAULT_LIMIT,
-  enabled: boolean = true,
-  referrer: string = 'use-indexed-spans'
-) => {
+export const useIndexedSpans = ({
+  filters,
+  sorts = [{field: 'timestamp', kind: 'desc'}],
+  limit = DEFAULT_LIMIT,
+  enabled = true,
+  referrer = 'use-indexed-spans',
+  fields = undefined,
```

Comment:
Highly recommend stealing the `Fields` typing from `useSpanMetrics` to get exact type safety on the data that comes back

Reply:
I took a look at `Fields` from `useSpanMetrics` and I'm not really sure we can safely use it? Theres a bunch of fields that don't overlap with `SpanIndexedField`. I updated to just use `SpanIndexedField` for now since it seems more specific to what we want in `useIndexedSpans`.

If there's supposed to be some relationship between  `SpanIndexedField` and `MetricsProperty`, we probably need to clean this up to make it more clear in a future pr.

lmk if this looks good to you!

Reply:
Okay that's fine, don't sweat it